src/Controller/GameController.php

```diff
@@ -48,113 +52,89 @@ public function notify(PublisherInterface $publisher, UuidV4 $gameId, array $par
     }
 
     /**
-     * @Route("/game/create", name="newGame")
+     * @Route("/games", name="game_create", methods={"POST"})
      */
-    public function newGame(): Response
+    public function newGame(request $request): Response
     {
 
-        $entityManager = $this->getDoctrine()->getManager();
-        $playerHash = $this->playerService->generatePlayerHash();
-        $game = $this->gameService->initGame($playerHash);
+        $content = $request->toArray();
+        if (!$content || !$content["playerId"]) {
+            return new JsonResponse("Id is required", 400);
+        }
+        $playerId = $content["playerId"];
 
+        $playerRepository = $this->getDoctrine()->getRepository(Player::class);
+        $player = $playerRepository->find($playerId);
+
+        
+        $game = $this->gameService->initGame($player);
+        
+        $entityManager = $this->getDoctrine()->getManager();
         $entityManager->persist($game);
         $entityManager->flush();
 
-        $response = $this->redirectToRoute(
-            'game',
-            [
-                "id" => $game->getId()
-            ]
-        );
-        $response->headers->setCookie(new Cookie($this::COOKIE_KEY, $playerHash));
-
+        $response = new JsonResponse(new GameDto($game));
+        $response->headers->set('Access-Control-Allow-Origin', '*');
         return $response;
     }
 
     /**
-     * @Route("/games/{id}/join", name="joinGame")
+     * @Route("/games/{id}/join", name="game_join", methods={"POST"})
      */
     public function joinGame(Request $request, string $id, PublisherInterface $publisher): Response
     {
-        $playerHash = $request->cookies->get($this::COOKIE_KEY);
+        $content = $request->toArray();
+        if (!$content || !$content["playerId"]) {
+            return new JsonResponse("Id and playerId are required", 400);
```

Comment:
This is in line with the previous comment on CORS. Here, as you return an answer, won't you miss the CORS?
In my opinion, CORS should be treated in a more generic way.